### PR TITLE
Fix bug and update TCTS.

### PR DESCRIPTION
**Pull request by lwwang1995**

### CODE REVIEW

**Comment by you-n-g** [comment]
@lwwang1995  Please resolve the conflictions

File: examples/benchmarks/TCTS/workflow_config_mtl_Alpha360_3correlated_task1.yaml [hunk 1]
```diff
@@ -0,0 +1,89 @@
+qlib_init:
```
Comment (you-n-g) [inline examples/benchmarks/TCTS/workflow_config_mtl_Alpha360_3correlated_task1.yaml:1]: Where is this file used?

**Comment by dingtine** [comment]
ERROR - qlib.workflow - [utils.py:38] - An exception has been raised[TypeError: expected str, bytes or os.PathLike object, not TFTModel].
  File "/Library/Frameworks/Python.framework/Versions/3.8/bin/qrun", line 8, in <module>
    sys.exit(run())
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/qlib/workflow/cli.py", line 62, in run
    fire.Fire(workflow)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/fire/core.py", line 141, in Fire
    component_trace = _Fire(component, args, parsed_flag_args, context, name)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/fire/core.py", line 466, in _Fire
    component, remaining_args = _CallAndUpdateTrace(
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/fire/core.py", line 681, in _CallAndUpdateTrace
    component = fn(*varargs, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/qlib/workflow/cli.py", line 56, in workflow
    recorder = task_train(config.get("task"), experiment_name=experiment_name)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/qlib/model/trainer.py", line 101, in task_train
    recorder = end_task_train(recorder, experiment_name)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/qlib/model/trainer.py", line 65, in end_task_train
    R.save_objects(**{"params.pkl": model})
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/qlib/workflow/__init__.py", line 449, in save_objects
    self.get_exp().get_recorder().save_objects(local_path, artifact_path, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/qlib/workflow/recorder.py", line 312, in save_objects
    Serializable.general_dump(data, path)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/qlib/utils/serial.py", line 140, in general_dump
    obj.to_pickle(path)
  File "/Users/tine/Downloads/qlib-fix_model_bug/examples/benchmarks/TFT/tft.py", line 315, in to_pickle
    super().to_pickle(self, path / "qlib_model")
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/qlib/utils/serial.py", line 87, in to_pickle
    with Path(path).open("wb") as f:
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/pathlib.py", line 1042, in __new__
    self = cls._from_parts(args, init=False)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/pathlib.py", line 683, in _from_parts
    drv, root, parts = self._parse_args(args)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/pathlib.py", line 667, in _parse_args
    a = os.fspath(a)
TypeError: expected str, bytes or os.PathLike object, not TFTModel

还是有点bug

**Comment by you-n-g** [comment]
@dingtine   It looks like you are running a TFT model instead of TCTS.

**Pull request closed without merging**